ui/apps/platform/cypress/integration/integrations/signatureIntegrations.test.js

```diff
@@ -126,6 +126,15 @@ I8+UmQtwa0MOOcoUeXXJXjGagodO6A22hzjwQyf5e87eeLA1FfwtGYNLjoA=
         );
         cy.get(selectors.buttons.save).should('be.disabled');
 
+        // Check conditional states.
+        getInputByLabel('Enable transparency log validation').uncheck();
+        getInputByLabel('Enable certificate transparency log validation').uncheck();
+
+        getHelperElementByLabel('Rekor URL').should('be.disabled');
```

Comment:
Is `getHelperElementByLabel` a typo? The assertion fails for all 3 integration test environments?

Reply:
Yep, mixed it up with `getInputByLabel`.